ch04.adoc

```diff
@@ -236,25 +236,24 @@ UDUNITS permits a number of alternatives to the word **`since`** in the units of
 All the alternatives have exactly the same meaning in UDUNITS.
 For compatibility with other software, CF strongly recommends that `since` should be used.
 
-The reference datetime string (appearing after the identifier **`since`**) is required.
-It may include date alone, or date and time, or date, time and time zone offset.
-Its format is __y__-__m__-__d__ [__H__:__M__:__S__ [__Z__]], where [...] indicates an optional element,
+The reference datetime string (appearing after the identifier **`since`**) is required. It must include the date, which may optionally be followed by time or time zone offset or both. Its format is __y__-__m__-__d__ [__H__:__M__:__S__ [__Z__]], where [...] indicates an optional element:
```

Comment:
This is maybe a bit pedantic (that's why we're here, though :)), but https://en.wikipedia.org/wiki/ISO_8601) implies that there is no space before a `Z`. I.e. in the schema it describes no space, and no space is used in any of the examples. Should we omit the space, here, too? This would also apply to the changes made in chapter 7.

Reply:
Our format is UDUNITS,  not ISO. You can have a space before _Z_ in UDUNITS. In fact you must have space if _Z_ is a number like 6 e.g. "days since 2025-01-01 16:00 6", which means the same as "days since 2025-01-01 16:00+6" and "days since 2025-01-01 16:00 +6". If _Z_ is `Z` or begins with `+` or `-` the space is optional.

Reply:
The syntax should be __y__-__m__-__d__ [__H__:__M__:__S__] [__Z__] i.e. the _Z_ is not inside the [ ] for time, because you can include _Z_ even if the time is omitted, as the modified text says. Also, as a matter of style in the asciidoc, each sentence starts a new line (unlike in markdown, inconveniently).

Reply:
Done

(and I think the each sentence starts with a newline thing is to make it more git-compatible, since git deals in lines. a while paragraph with no newlines make git diffs hard to figure out.)

Reply:
Good for me.